packages/astro/test/astro-attrs.test.js

```diff
@@ -25,32 +26,32 @@ describe('Attributes', () => {
 
     for (const [k, v] of Object.entries(attrs)) {
       const attr = $(`#${k}`).attr('attr');
-      expect(attr).toBe(v);
+      expect(attr).to.equal(v);
     }
   });
 
-  test('Passes boolean attributes to components as expected', async () => {
+  it('Passes boolean attributes to components as expected', async () => {
     const html = await fixture.readFile('/component/index.html');
     const $ = cheerio.load(html);
 
-    expect($('#true').attr('attr')).toBe('attr-true');
-    expect($('#true').attr('type')).toBe('boolean');
-    expect($('#false').attr('attr')).toBe('attr-false');
-    expect($('#false').attr('type')).toBe('boolean');
+    expect($('#true').attr('attr')).to.equal('attr-true');
```

Comment:
Mocha + Chai was selected because of its similarity to Jest. Migrating was just a quick find-and-replace.